pandas/io/json/normalize.py

```diff
@@ -100,7 +114,9 @@ def json_normalize(data, record_path=None, meta=None,
                    meta_prefix=None,
                    record_prefix=None,
                    errors='raise',
-                   sep='.'):
+                   sep='.',
+                   max_level=None,
```

Comment:
Unless there's a reason for this to be `None` probably simplifies code if this is 0, no?

Reply:
`None` for the case where you don't set a `max_level` and the nesting happens at all levels.